### PR TITLE
updated test description

### PR DESCRIPTION
**Pull request by mringler**
Stumbled over the documentation for tests and found it a bit convoluted and outdated in a lot of places. Hope this works better!

### CODE REVIEW

File: documentation/cookbook/working-with-test-suite.markdown [hunk 1]
```diff
@@ -346,14 +199,34 @@ public function testWholeWorld(){}
...
+composer run-script stan
+composer run-script psalm
+```
+
+### Code style ###
+
+You can fix code style before creating a PR using [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer) and the provided project configuration.
+
+```bash
+composer run-script cs-fix # automatically adjusts formatting
```
Comment (dereuromark) [inline documentation/cookbook/working-with-test-suite.markdown:219]: composer cs-fix 
etc suffices

Comment (mringler) [reply documentation/cookbook/working-with-test-suite.markdown]: oh man, and I have been typing it out for years like an idiot lol

**Comment by mringler** [comment]
Hmpf, tables are not rendering at http://propelorm.org/documentation/cookbook/working-with-test-suite.html
I'm looking into it

**Comment by mringler** [comment]
PR to fix tables at #432

**Pull request merged**